src/app/components/Forms/PopupForm.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import styled from 'styled-components'
+import Button from '../Button/Button'
+import OverlayWrapper from '../OverlayWrapper/OverlayWrapper'
+
+type FormProps = {
+  onCancel: () => void
+  onSubmit: (event: React.FormEvent<HTMLFormElement>) => void
+  SubmitText: string
+  children: React.ReactNode
+}
+
+export default function Form({
+  onCancel,
+  onSubmit,
+  children,
+  SubmitText,
+}: FormProps): JSX.Element {
+  return (
+    <OverlayWrapper onReturn={onCancel}>
+      <FormContainer onSubmit={onSubmit}>
+        <FormTitle>{SubmitText}</FormTitle>
+        {children}
+        <ButtonContainer>
+          <Button type="button" onClick={onCancel}>
+            Cancel
+          </Button>
+          <Button highlight>{SubmitText}</Button>
+        </ButtonContainer>
+      </FormContainer>
+    </OverlayWrapper>
+  )
+}
+const FormTitle = styled.h2`
+  font-size: 2.8rem;
+  text-transform: uppercase;
+`
+
+const FormContainer = styled.form`
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding: 20px;
+  color: rgba(255, 255, 255, 0.3);
```

Comment:
Modern `rgb`
https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb()

```suggestion
  color: rgb(255 255 255 / 30%);
```